test/e2e/pipeline/pipeline_test.go

```diff
@@ -284,10 +284,6 @@ Waiting for logs to be available...
 	kubectl.MustSucceed(t, "create", "-f", helper.GetResourcePath("pipeline-with-workspace.yaml"))
 
 	t.Run("Start PipelineRun with --workspace and volumeClaimTemplate", func(t *testing.T) {
-		if tkn.CheckVersion("Pipeline", "v0.10.2") {
```

Comment:
It seems this runs appropriately without the version check. Are we still running these tests against a cluster with v0.10.2 installed?

Reply:
Addressed by https://github.com/tektoncd/plumbing/pull/596. Will add this back.